cmd/create_test.go

```diff
@@ -194,7 +210,7 @@ func (mcc *createCmd) mockDetectLanguage() (*config.DraftConfig, string, error)
 			mcc.createConfig.LanguageType = mcc.lang
 		} else {
 			langs, err = linguist.ProcessDir(mcc.dest)
-			log.Debugf("linguist.ProcessDir(%v) result:\n\nError: %v", mcc.dest, err)
```

Comment:
any reason for err removal in the debug log?

Reply:
I inserted that in a previous commit to solve an issue. I can leave it there though